python_modules/dagster-graphql/dagster_graphql/implementation/loader.py

```diff
@@ -428,7 +428,7 @@ def get_sink_asset(self, asset_key: AssetKey) -> ExternalAssetNode:
 
     def get_cross_repo_dependent_assets(
         self, repository_location_name: str, repository_name: str, asset_key: AssetKey
-    ) -> List[ExternalAssetDependedBy]:
+    ) -> Sequence[ExternalAssetDependedBy]:
```

Comment:
I feel like a lot of this stuff is unrelated?

Reply:
Sorry you caught it when I was in the middle of fixing my screwed up graphite state so it was reflecting downstack PRs-- now updated